proto/process/agent.proto

```diff
@@ -784,10 +784,12 @@ message Cluster {
 	uint64 memoryCapacity = 7;
 	uint64 cpuAllocatable = 8;
 	uint64 cpuCapacity = 9;
-	string resourceVersion = 10;
```

Comment:
The field numbers are used to encode and decode the proto message, so you typically don't want to change any previously assigned proto field numbers. This would break backward compatibility.

See https://protobuf.dev/programming-guides/proto3/#assigning for more info

Reply:
Fixed, thank you.